code/modules/research/designs/weapons.dm

```diff
@@ -20,7 +20,7 @@
 	category = "Weapons"
 	build_path = /obj/item/weapon/gun/energy/stunrevolver
 	locked = 1
-	req_lock_access = list(access_security)
+	req_lock_access = list(access_security, access_weapons)
```

Comment:
It doesn't, I changed the lockbox access checks to use req_one_access, which seems like it was what was intended at some point. The temperature gun needed (access_tox, access_robotics, access_rd), which doesn't really make sense when using req_access.